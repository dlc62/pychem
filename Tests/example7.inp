[example7]
Method = "HF"
Basis_Sets = ["STO-3G"]
Multiplicity = 1
Charge = 0
Coords_Units = "BOHR"
Coords = [['B', 5.0,  0.  ,  0.,   0.],
          ['H', 1.0, 3.013923978561,   0.,   0.]]
Reference = "RHF"
Max_SCF_Iterations = 50
Job_Type = "PROPERTY"
Property_Type = "SCATTERING"
Property_Grid = [0.0]

