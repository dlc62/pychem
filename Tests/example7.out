-------------------------------------------------------
 Hartree-Fock iterations 
-------------------------------------------------------

 Cycle:  1
 Total energy:  -23.7581533611
 Change in energy:  12.1356299693
 DIIS error:  0.232146481581

 Cycle:  2
 Total energy:  -24.6159262732
 Change in energy:  -0.857772912133
 DIIS error:  0.061471588104

 Cycle:  3
 Total energy:  -24.7405278106
 Change in energy:  -0.124601537377
 DIIS error:  0.0200174296669

 Cycle:  4
 Total energy:  -24.7004014346
 Change in energy:  0.040126375973
 DIIS error:  0.00129850280294

 Cycle:  5
 Total energy:  -24.7030955236
 Change in energy:  -0.00269408893046
 DIIS error:  4.59827430766e-05

 Cycle:  6
 Total energy:  -24.7030870786
 Change in energy:  8.44495938424e-06
 DIIS error:  7.65614675429e-07

 Cycle:  7
 Total energy:  -24.7030877843
 Change in energy:  -7.05728066208e-07
 DIIS error:  1.05663702565e-07

-------------------------------------------------------
 End of Hartree-Fock iterations 
-------------------------------------------------------

Final HF energy:  -24.7030877843


 Alpha Orbital Energies 
[-7.36692302 -0.4902569  -0.24531783  0.25754157  0.25754157  0.45390037]

 Alpha MOs 
[[  9.92556652e-01   2.44373304e-01  -1.64730630e-01   4.79860309e-16   8.22654686e-18  -1.00299525e-01]
 [  2.69256588e-02  -7.87747097e-01   6.24036498e-01  -1.90545609e-15  -1.52153606e-17   4.30302461e-01]
 [  3.76276203e-03  -1.33590594e-01  -5.75138605e-01   1.20007847e-16   1.33159083e-17   8.91905038e-01]
 [  2.04439207e-17  -4.73208658e-16  -2.53430267e-15  -9.98823383e-01   4.84953704e-02  -1.67999370e-15]
 [  9.79002580e-18   3.81921847e-18  -1.14511728e-16  -4.84953704e-02  -9.98823383e-01  -5.13316622e-17]
 [ -4.52002364e-03  -4.22833708e-01  -5.37655628e-01   3.16865367e-15  -6.94614897e-18  -8.86721136e-01]]

 Alpha Density Matrix 
[[  1.07202320e+00  -2.68577045e-01   6.58317242e-02   3.22129461e-16   2.95140594e-17  -1.92472997e-02]
 [ -2.68577045e-01   1.01069203e+00  -2.53570563e-01  -1.20817815e-15  -7.42044731e-17  -2.55241326e-03]
 [  6.58317242e-02  -2.53570563e-01   3.48645020e-01   1.52086845e-15   6.53867414e-17   3.65696106e-01]
 [  3.22129461e-16  -1.20817815e-15   1.52086845e-15   6.64703440e-30   2.88600237e-31   1.56257826e-15]
 [  2.95140594e-17  -7.42044731e-17   6.53867414e-17   2.88600237e-31   1.32233669e-32   5.99087296e-17]
 [ -1.92472997e-02  -2.55241326e-03   3.65696106e-01   1.56257826e-15   5.99087296e-17   4.67882350e-01]]


-------------------------------------------------------
 End of Hartree-Fock calculation 
-------------------------------------------------------

-------------------------------------------------------
 Property calculation - electron scattering intensities 
-------------------------------------------------------

Grid value -> Scattering patterns for each electronic state 

  0.000000 36.000000000000

-------------------------------------------------------
 End of property calculation 
-------------------------------------------------------

